.github/workflows/auto_updates.yml

```diff
@@ -48,14 +48,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: 'poetry'
 
-      # TODO: Use a "CI" group to ensure only the dependencies needed for running in the CI environment are installed.
-      #       This should be possible when Poetry 1.2 is available, using the groups feature:
-      #       https://github.com/python-poetry/poetry/issues/1644
-      #       https://github.com/phylum-dev/phylum-ci/issues/15
       - name: Install the project with poetry
         run: |
           poetry env use python${{ matrix.python-version }}
-          poetry install --verbose --no-root
+          poetry install --verbose --no-root --sync
```

Comment:
I don't see any `--with ci` here, shouldn't that be supplied for CI dep installation?

Reply:
In this case, for the `auto_updates` workflow, no `--with` option is needed. In fact, it turns out that this step is not needed at all. That is, the `poetry update` command...which is the real attraction in this workflow...does not require the project to be installed at all. It will resolve and update _all_ of the dependencies specified in `pyproject.toml`, regardless of whether they are part of the implicit `main` group or another named dependency group. Update coming...